lib/cretonne/meta/semantics/smtlib.py

```diff
@@ -0,0 +1,150 @@
+"""
+Tools to emit SMTLIB bitvector queries encoding concrete RTLs containing only
+primitive instructions.
```

Comment:
I can file a followup issue to change this. It would definitely be cleaner to depend on z3's python bindings.